solutions/drmmessages/drmmessages.py

```diff
@@ -0,0 +1,40 @@
+"""
+https://open.kattis.com/problems/drmmessages
+
+Author: Sam Westigard
+"""
+from sys import stdin
+
+def rotationValue(string):
+  total = sum([ord(i)-65 for i in string])
+  return total
+
+def rotateLetter(letter, rotations):
+  alphabet = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']
```

Comment:
I think it's better to use `chr(ord('A') + offset)` instead of hard-coding the characters.